src/store/models/network.spec.ts

```diff
@@ -483,6 +488,7 @@ describe('Network model', () => {
     beforeEach(() => {
       const { addNetwork } = store.getActions().network;
       addNetwork(addNetworkArgs);
+      mockDetectPort.mockImplementation(port => Promise.resolve(port));
```

Comment:
I'm not following why you need to change the mock's implementation in these tests in this file. They succeed without this.

Reply:
Removed.